Classes/Systems/Toaster.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Toaster.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 5/17/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import Toaster
+
+enum Toaster {
+
+    private static func provideHapticFeedback() {
+        UINotificationFeedbackGenerator().notificationOccurred(.error)
+    }
+
+    static func showRevokeError() {
+        showError(message: NSLocalizedString("Your access token was revoked.", comment: ""))
+    }
+
+    static func showNetworkError() {
+        showError(message: NSLocalizedString("Cannot connect to GitHub.", comment: ""))
+    }
+
+    static func showGenericError() {
+        showError(message: NSLocalizedString("Something went wrong.", comment: ""))
+    }
+
+    static func showPermissionsError() {
+        showError(message: NSLocalizedString("You must request access.", comment: ""))
+    }
+    
+    static func showError(message: String,
+                          textColor: UIColor = UIColor.white,
+                          backgroundColor: UIColor = Styles.Colors.Red.medium.color ) {
+        let toast = Toast(text: message, duration: Delay.long)
+        var bottomInset: CGFloat = 0.0
+
+        if #available(iOS 11, *) {
+            let insets = UIApplication.shared.delegate?.window??.safeAreaInsets
+
+            bottomInset = insets?.bottom ?? 0
+        }
```

Comment:
We need to handle iOS <11 in the `else` right?

Reply:
The only reason I use the if available is to get the iPhone X safe area. 

If it is a iPhone 8 or 7 etc the insets?.bottom will be 0. So the same as pre iOS 11